src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -97,7 +97,7 @@ void sl_ble_init()
     rsi_ble_app_init_events();
 
     SILABS_LOG("StartAdvertising");
```

Comment:
You have a print for start Advertising here and also in the drive ble state

I'd remove this one. 

Also use the chip logging API not SILABS_LOG, even more so for non app specific logging

Reply:
Removed